packages/datepicker/src/react/sub-field.tsx

```diff
@@ -0,0 +1,62 @@
+import { combineFns, ValueOf } from '@pluralsight/ps-design-system-util'
+import { compose, css } from 'glamor'
+import React, { FC, useRef } from 'react'
+
+import stylesheet from '../css'
+import * as vars from '../vars'
+
+const styles = {
+  subField: (appearance: ValueOf<typeof vars.appearances>) =>
+    compose(
+      css(stylesheet['.psds-date-picker__sub-field']),
+      css(stylesheet[`.psds-date-picker__sub-field--appearance-${appearance}`])
+    ),
+  subFieldDivider: (appearance: ValueOf<typeof vars.appearances>) =>
+    compose(
+      css(stylesheet['.psds-date-picker__sub-field-divider']),
+      css(
+        stylesheet[
+          `.psds-date-picker__sub-field-divider--appearance-${appearance}`
+        ]
+      )
+    )
+}
+
+interface SubFieldProps
+  extends Omit<
+    React.HTMLAttributes<HTMLInputElement>,
+    'onBlur' | 'onChange' | 'onFocus'
+  > {
+  appearance: ValueOf<typeof vars.appearances>
+  disabled?: boolean
+  name: string
+  onBlur: (evt: React.FocusEvent) => void
+  onFocus: (evt: React.FocusEvent) => void
+  value: number | undefined
```

Comment:
This can be optional as well `value?: number`

Reply:
if the explanation above for the DatePickerProps passes muster, I think it'll apply here too.  Let me know.